experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl

```diff
@@ -1212,6 +1234,9 @@ error.
 julia> qsm_model = literature_model(arxiv_id = "1903.00009", model_parameters = Dict("k" => 4))
 Hypersurface model over a concrete base
 
+julia> h = euler_characteristic(qsm_model; check = false)
+378
+
```

Comment:
```suggestion
```
these three lines are already there right below

Reply:
I added these lines twice on purpose to function as an additional test

Reply:
Just to elaborate a bit: Once `euler_characteristic` is called for the 2nd time, it fetches the saved value of the attribute. This process failed in the past, because of the wrong type cast. Now this will be tested.